service/web_api.py

```diff
@@ -195,6 +195,33 @@ def check_model_exist():
     return jsonify({"code": 0, "message": "success", "exists": result_list})
 
 
+@app.route("/api/checkURLExists", methods=["POST"])
+def check_url_exists():
+    address = request.get_json()
```

Comment:
the variable switches between address and repo_id. From reading the code, it should be the same everywhere.
From what I understand, the value is derived directly from the text input field and thus should be address (or the correct terminus for full model path in hugging face (is it repo_id?))